### DISCUSSION TITLE
How to github integrate with a branch instead of master?

### DISCUSSION THREAD

**Discussion by joshatt** [vercel/vercel#4454] [Help] [answered]
Github integration......  with a branch instead of master? I googled and did not find the answer. 
Could anybody please help?

**Comment by idjevm** [comment, marked as answer]
To be able to deploy when pushed to a specific branch, add your branch in this setting ->  

`https://vercel.com/{username}/{project_name}/settings/domains`

You should see the following screen: 
![image](https://user-images.githubusercontent.com/25408660/82635855-3b152b00-9bcf-11ea-8146-463f440f7eda.png)

Here, you can change the Git Branch to an existing branch on your project. 

Note that this means that any pushes to the selected branch will be at the domain that is on this screen. 

Let me know if you were able to figure out, as I gave it a quick test, and it worked for me. 

Make sure the selected branch exists within your Github project.

**Reply by joshatt** [reply]
> To be able to deploy when pushed to a specific branch, add your branch in this setting ->
> 
> `https://vercel.com/{username}/{project_name}/settings/domains`
> 
> You should see the following screen:
> ![image](https://user-images.githubusercontent.com/25408660/82635855-3b152b00-9bcf-11ea-8146-463f440f7eda.png)
> 
> Here, you can change the Git Branch to an existing branch on your project.
> 
> Note that this means that any pushes to the selected branch will be at the domain that is on this screen.
> 
> Let me know if you were able to figure out, as I gave it a quick test, and it worked for me.
> 
> Make sure the selected branch exists within your Github project.

Thank you, sir, that works perfectly well.  I appreciate it.
